src/schema/schema.generated.ts.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/schema/schema.generated.ts b/src/schema/schema.generated.ts
+index 941e952..927e2c6 100644
+--- a/src/schema/schema.generated.ts
++++ b/src/schema/schema.generated.ts
+@@ -614,9 +614,7 @@ export interface RowSet {
+       }[]
+     | null;
+ }
+-export interface RowFieldValue {
+-  [k: string]: unknown;
+-}
++export type RowFieldValue = unknown; // Manually corrected :(
+ export interface MutationRequest {
+   /**
+    * The mutation operations to perform
```

Comment:
What is this patch used for again?

Reply:
This is the manual patch to the generated types that was introduced in commit https://github.com/hasura/ndc-sdk-typescript/commit/dec3d34f5cf12a0b95f9934f77e745242c05b473 but is now just formalised and automated into this file instead of being an easily lost change to a generated file.

Reply:
Ah right that makes sense.